test/recipe/admin/.baked/index.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$type": "Record",
+  "title": "Dashboard",
+  "path": "",
+  "props":[
+    {
+      "name": "menu",
+      "component": {
+        "$type": "Menu",
+        "title": "Menu",
+        "path":""
+      }
+    }
+  ],
+  "data":{
+    "menu": [
+      {
+        "title": "Rich Transient w/ Datas 1",
+        "path": "/rich-transient-with-datas/test1"
+      },
+      {
+        "title": "Rich Transient w/ Datas 2",
+        "path": "/rich-transient-with-datas/test2"
+      },
+      {
+        "title": "Rich Transient w/ Datas 3",
+        "path": "/rich-transient-with-datas/test3"
+      }
+    ]
+  }
```

Comment:
we need to refactor the component to have this near `"$type": "Menu"` declaration under `"component"`.